packages/platform-ios/native_modules.rb

```diff
@@ -5,17 +5,28 @@
 #
 def use_native_modules!(root = "..", packages = nil)
   if (!packages)
-    # Resolve the CLI's main index file
-    cli_bin = Pod::Executable.execute_command("node", ["-e", "console.log(require.resolve('@react-native-community/cli/build/index.js'))"], true).strip
+    command = "node"
+    args = ["./node_modules/.bin/react-native", "config"]
+    begin
+      # Check if project uses Yarn
+      Pod::Executable.execute_command("node", ["-e", "console.log(require.resolve('#{root}/yarn.lock'))"], true)
+      command = "yarn"
+      args = ["run", "react-native", "config"]
```

Comment:
Protip: If you run `yarn run --silent react-native config` you shouldn't get the headers and footers 🙂

Reply:
Thanks! I tried that and was pretty sure I've still seen the headers/footers with `-s` flag, but clearly must have missed it, because it works now :D